node/src/BaseClient.ts

```diff
@@ -3694,6 +3695,31 @@ export class BaseClient {
         return this.createWritePromise(scriptInvocation, options);
     }
 
+    /**
+     * Returns the original source code of a script in the script cache.
+     *
+     * @see {@link https://valkey.io/commands/script-show|valkey.io} for more details.
+     * @remarks Since Valkey version 8.0.0.
+     *
+     * @param sha1 - The SHA1 digest of the script.
+     * @param options - (Optional) See {@link DecoderOption}.
+     * @returns The original source code of the script, if present in the cache.
+     * @throws RequestError if the script is not found in the cache.
+     *
+     * @example
+     * ```typescript
+     * const scriptHash = script.getHash();
+     * const scriptSource = await client.scriptShow(scriptHash);
+     * console.log(scriptSource); // Output: "return { KEYS[1], ARGV[1] }"
+     * ```
+     */
+    public async scriptShow(
+        sha1: GlideString,
```

Comment:
nit: is there a reason why we don't pass the Script object and internally we can call script.getHash()?

Reply:
same as script exists, kill ect...

Reply:
well... the comment could apply to any of these commands.

Reply:
I have no idea 🙉